### PR TITLE
feat: select field

### LINKED ISSUE #45: Select component
- Dropdown select box

### PR DESCRIPTION
**Pull request by belohlavek**
Screen Shot 2019-09-02 at 09 55 26

Screen Shot 2019-09-02 at 11 54 15

Closes #45

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import 'react-tile-map/lib/styles.css'
 import './themes/base-theme.css'
 
 // Default theme
-import './themes/alternative/light-theme.css'
+import './themes/alternative/dark-theme.css'
```
Comment (belohlavek) [inline src/index.ts]: boo

**Comment by belohlavek** [comment]
Missing styles on light version

File: src/components/SelectField/SelectField.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import './SelectField.css'
+
+export type SelectFieldProps = DropdownProps & {
+  label?: string
+}
+
+export class SelectField extends React.PureComponent<SelectFieldProps> {
+  hasAction() {
+    let { loading, error, action, onAction } = this.props
+    return !this.isAddress() && !loading && !error && action && onAction
```
Comment (nicosantangelo) [inline src/components/SelectField/SelectField.tsx]: `action` and `onAction` are not being used in this component, I think this comes from Field?
It's a bit weird that the linter does not complain

File: src/components/SelectField/SelectField.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import { Header, Dropdown, DropdownProps } from 'semantic-ui-react'
+import './SelectField.css'
+
+export type SelectFieldProps = DropdownProps & {
+  label?: string
+}
+
+export class SelectField extends React.PureComponent<SelectFieldProps> {
+  hasAction() {
+    let { loading, error, action, onAction } = this.props
```
Comment (nicosantangelo) [inline src/components/SelectField/SelectField.tsx]: This is a bit of a stretch but most of the code is using `const` for everything except when a variable is re-assigned. I know Field.tsx does the same thing for this method, but maybe we can change both?
Same for `let { label, ...rest } = this.props`

**Pull request merged**